From 888f675a6e5b3322a45c06ca2162663edd7cd7b3 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Wed, 24 Jul 2019 20:08:23 +0100
Subject: [PATCH 2/3] d-m-{merge,debrebase}(7): Set 'merge' in gbp.conf

This enables simplifying runes which are more frequently to be typed.

'merge-mode' no longer need be set since in the only case where the
manpages recommend allowing `gbp import-orig` to perform a merge, a
different merge mode is required (and specified in that rune).

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 8 ++++----
 dgit-maint-merge.7.pod     | 7 ++++---
 2 files changed, 8 insertions(+), 7 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index 2ac70b57..28406697 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -188,7 +188,7 @@ Now create I<debian/gbp.conf>:
     pristine-tar-commit = False
 
     [import-orig]
-    merge-mode = merge
+    merge = False
 
 =back
 
@@ -208,7 +208,7 @@ Then we can import the upstream version:
 
 =over 4
 
-    % gbp import-orig --merge-mode=replace ../foo_1.2.2.orig.tar.xz
+    % gbp import-orig --merge --merge-mode=replace ../foo_1.2.2.orig.tar.xz
 
 =back
 
@@ -372,7 +372,7 @@ Then, either
 
 =over 4
 
-    % gbp import-orig --no-merge ../foo_1.2.3.orig.tar.xz
+    % gbp import-orig ../foo_1.2.3.orig.tar.xz
 
 =back
 
@@ -380,7 +380,7 @@ or if you have a working watch file
 
 =over 4
 
-    % gbp import-orig --no-merge --uscan
+    % gbp import-orig --uscan
 
 =back
 
diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index 17637bad..71cb508c 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -174,6 +174,7 @@ Now create I<debian/gbp.conf>:
 
     [import-orig]
     merge-mode = merge
+    merge = False
 
 =back
 
@@ -193,7 +194,7 @@ Then we can import the upstream version:
 
 =over 4
 
-    % gbp import-orig --merge-mode=replace ../foo_1.2.2.orig.tar.xz
+    % gbp import-orig --merge --merge-mode=replace ../foo_1.2.2.orig.tar.xz
 
 =back
 
@@ -381,7 +382,7 @@ Then, either
 
 =over 4
 
-    % gbp import-orig --no-merge ../foo_1.2.3.orig.tar.xz
+    % gbp import-orig ../foo_1.2.3.orig.tar.xz
 
 =back
 
@@ -389,7 +390,7 @@ or if you have a working watch file
 
 =over 4
 
-    % gbp import-orig --no-merge --uscan
+    % gbp import-orig --uscan
 
 =back
 
-- 
2.20.1

